lib/modules/manager/cake/__fixtures__/build.cake

```diff
@@ -1,7 +1,7 @@
 foo
 #addin nuget:?package=Foo.Foo
 #addin "nuget:?package=Bim.Bim&version=6.6.6"
-#tool nuget:https://example.com?package=Bar.Bar&version=2.2.2
```

Comment:
please add this case back to test both

Reply:
done. 

I also added a test for an url that ends in a file name - did not think about adding that before.